tests/core/test_data.py

```diff
@@ -158,7 +159,7 @@ def test_find_nlu_files_with_different_formats(test_input, expected):
     examples_dir = "data/examples"
     data_dir = os.path.join(examples_dir, test_input)
     core_files, nlu_files = data.get_core_nlu_files([data_dir])
-    assert nlu_files == expected
+    assert [Path(f) for f in nlu_files] == [Path(f) for f in expected]
```

Comment:
why did we need to change that?

Reply:
is it because the `relative_normpath` function changed?

Reply:
Exactly. Because it doesn't return string anymore, but `Path` instead.